README.md

```diff
@@ -536,7 +536,48 @@ We also add a `upgraded_from_workspace_id` property to the upgraded object, to i
 
 The upgrade process can be triggered using the `migrate-tables` [UCX command](#migrate-tables-command)
 
-Or by running the "Migrate Tables" workflow deployed to the workspace.
+Or by running the table migration workflows deployed to the workspace.
+
+##### table migration workflows
```

Comment:
Move this to the top or end of the `Table Migration` process to show an overview to users.

```suggestion
#### Table migration workflows
```